includes/definitions.inc.php

```diff
@@ -888,8 +888,8 @@
 $config['os'][$os]['over'][2]['text']  = 'Memory Usage';
 
 $os = 'fabos';
-$config['os'][$os]['text']             = 'Brocade FabricOS';
-$config['os'][$os]['type']             = 'network';
+$config['os'][$os]['text']             = 'FabricOS';
+$config['os'][$os]['type']             = 'storage';
```

Comment:
still, being a "switch" it's not making it networking. maybe the categories are too loose in definition and this is way we are having this discussion :) in my opinion FC switches (that cannot provide ethernet connectivity) are storage infrastructure, not networking.